packages/cli/src/lib/missing-routes.ts

```diff
@@ -30,12 +30,16 @@ const REQUIRED_ROUTES = [
 
   'account',
   'account/login',
-  'account/register',
   // 'account/addresses',
   // 'account/orders',
   'account/orders/:orderId',
-  'account/reset/:id/:token',
-  'account/activate/:id/:token',
+  // -- Added for CAAPI:
+  'account/authorize',
+
+  // -- These were removed when migrating to CAAPI:
+  // 'account/register',
+  // 'account/reset/:id/:token',
+  // 'account/activate/:id/:token',
```

Comment:
Would it be bad to just delete these? Or is the comment valueable for historical reasons?